estilos/style.css

```
@@ -0,0 +1,95 @@
+@charset "UTF-8";
+
+* {
+    margin: 0px;
+    padding: 0px;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+    background-color: #B6B5F9;
+    
```

remova esta linha em branco.
```suggestion
    body {
         background-color: #B6B5F9;
     }
```